OM-Demo/AdListViewController.swift

```diff
@@ -70,22 +70,59 @@ class AdListViewController: UITableViewController {
     var adUnits: [AdUnit] = [.HTMLDisplay, .nativeVideo, .nativeDisplay, .nativeAudio]
 
     override func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
+        return 2
     }
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return adUnits.count
+        return section == 0 ? 1 : adUnits.count
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "listCell", for: indexPath)
-        let adUnit = adUnits[indexPath.row]
-        cell.textLabel?.text = adUnit.title
+        var cell = tableView.dequeueReusableCell(withIdentifier: "listCell", for: indexPath)
+        if indexPath.section == 0 {
+            cell = tableView.dequeueReusableCell(withIdentifier: "prerenderCell", for: indexPath)
+            cell.textLabel?.text = "Prerender"
+            
+            let toggle = UISwitch()
+            toggle.translatesAutoresizingMaskIntoConstraints = false
+            toggle.addTarget(self, action: #selector(togglePrerendering(sender:)), for: .touchUpInside)
```

Comment:
Not sure the right place (maybe in viewDidLoad?) we should set the toggle.isEnabled to the current value of `Settings.shared.isPrerendering` to make sure the UI remains in sync. Also, consider moving the UISwitch init/config step out of `tableView(_:cellForRowAt:)` implementation, perhaps to a property on AdListViewController?